regression-test/suites/nereids_syntax_p0/alias_conflict.groovy

```diff
@@ -0,0 +1,107 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+suite("alias_conflict") {
+
+    sql """ DROP TABLE IF EXISTS `test1` """
+    sql """ DROP TABLE IF EXISTS `test2` """
+    sql """ DROP TABLE IF EXISTS `test3` """
+
+    sql """
+        CREATE TABLE `test1` (
```

Comment:
`test1` table conflict to `nereids_syntax_p0/test_limit.groovy`, please rename to other table name, e.g. `test_alias_tbl1`, `test_alias_tbl2`